### PR TITLE
SAK-44718: SAMIGO - A lot of scroll on settings page

### PR DESCRIPTION
**Pull request by jesusmmp**

### CODE REVIEW

**Comment by ottenhoff** [comment]
What happens when you expand/open all settings panels? Is there a scroll bar after your patch?

**Comment by jesusmmp** [comment]
> What happens when you expand/open all settings panels? Is there a scroll bar after your patch?

All panels are expanded and the height is right. Yes, there is a scroll bar

File: samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp [hunk 1]
```diff
@@ -72,8 +72,6 @@
           $("#jqueryui-accordion-metadata").accordion({ heightStyle: "content",collapsible: true,active: false });
           // This is a sub-accordion inside of the Availability and Submission Panel
           $("#jqueryui-accordion-security").accordion({ heightStyle: "content",collapsible: true,active: false });
-          // adjust the height of the iframe to accomodate the expansion from the accordion
-          $("body").height($("body").outerHeight() + 900);
```
Comment (adrianfish) [inline samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp:76]: what happens when the accordion is expanded? Does it produce a scroll bar in the iframe, or does it cause the iframe to grow automatically?

**Review by adrianfish — CHANGES_REQUESTED** [review]

File: samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp (hunk 1 above)
Comment (jesusmmp) [reply samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp:76]: Now, there isn't any iframe. I answered the same question to @ottenhoff some days ago.

Comment (adrianfish) [reply samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp:76]: Sorry, didn't see that. Ok, cool, thanks.

**Review by adrianfish — APPROVED** [review]
GTG

**Pull request merged**